src/app/+orders/exports/order-form/order-form.component.ts

```diff
@@ -0,0 +1,267 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { AbstractControl, FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { AccountFacade } from 'facades/account.facade';
+import { CustomerFacade } from 'facades/customer.facade';
+import { ProductFacade } from 'facades/product.facade';
+import { Account } from 'models/account.model';
+import { Customer } from 'models/customer.model';
+import { OrderItem } from 'models/order-item.model';
+import { Order, OrderStatus } from 'models/order.model';
+import { Product } from 'models/product.model';
+import { BehaviorSubject, combineLatest, Observable, Subject } from 'rxjs';
+import { filter, map, shareReplay, startWith, take, takeUntil, withLatestFrom } from 'rxjs/operators';
+import { toCurrencyString } from 'shared/libs/currency.lib';
+import { OrderInfo } from 'src/app/+orders/models/order-info.model';
+import { FormGroupConfig } from 'src/app/shared/types/forms.type';
+
+@Component({
+  selector: 'app-order-form',
+  templateUrl: './order-form.component.html',
+  styleUrls: ['./order-form.component.scss']
+})
+export class OrderFormComponent implements OnInit, OnDestroy {
```

Comment:
We should revisit our components in the demo app in the future. I think we have an opportunity here to share with the Angular community ways of creating fully reactive apps built on NgRx as a platform of reactive programming. I think we could simplify our components considerably, centralize business logic into services, facades and more importantly...ngrx effects. For another day though, not necessary to change anything in this PR.